graphistry/compute/hop.py

```diff
@@ -251,9 +312,25 @@ def hop(self: Plottable,
         hop_edges_reverse = None
         new_node_ids_reverse = None
         if direction in ['reverse', 'undirected']:
+            # Prepare edges for reverse merging - handle column name conflicts
```

Comment:
All this logic looks similar to the case of `if direction in ['forward', 'undirected']`, maybe we can create a function to centralize the conflict resolution implementation.